public/app/features/scenes/variables/sets/SceneVariableSet.ts

```diff
@@ -97,6 +99,11 @@ export class SceneVariableSet extends SceneObjectBase<SceneVariableSetState> imp
    * Not sure if this should be handled here on in MultiValueVariable
    */
   private handleVariableError(variable: SceneVariable, err: Error) {
+    const update = this.updating.get(variable);
+    update?.subscription?.unsubscribe();
+
+    this.updating.delete(variable);
+    this.variablesToUpdate.delete(variable);
```

Comment:
@torkelo adding this to make sure query variable errors don't break the future updates of the variable.

Reply:
:+1: